src/main/java/bio/terra/cli/command/app/passthrough/Gcloud.java

```diff
@@ -15,4 +37,49 @@ public String getExecutableName() {
   public String getInstallationUrl() {
     return "https://cloud.google.com/sdk/docs/install";
   }
+
+  @Override
+  protected void executeImpl() {
+    workspaceOption.overrideIfSpecified();
+
+    if (bucketName != null && builds && submit) {
+      var resource = Context.requireWorkspace().getResource(bucketName);
+      if (Resource.Type.GCS_BUCKET != resource.getResourceType()) {
+        throw new UserActionableException(
+            String.format(
+                "%s %s cannot be cloned because it is not a gcs-bucket",
+                resource.getResourceType(), resource.getName()));
+      }
+      ArrayList<String> cloneCommands =
+          new ArrayList<>(
+              ImmutableList.of(
+                  "gcloud",
+                  "builds",
+                  "submit",
+                  "--async",
+                  "--timeout=2h",
+                  "--gcs-source-staging-dir=" + resource.resolve() + "/cloudbuild_source",
+                  "--gcs-log-dir=" + resource.resolve() + "/cloudbuild_logs",
+                  "--tag="
```

Comment:
I'm pretty sure Nicole doesn't want this --tag here. I just asked on PF-2053 exactly what she wants.